evaluate/train_image.py

```diff
@@ -68,6 +73,15 @@
 
 xp = cuda.cupy if args.gpu >= 0 else numpy
 
+if args.gpu >= 0:
+    free_mem, total_mem = cuda.cupy.cuda.runtime.memGetInfo()
```

Comment:
Please use `_` to if the variable is not used (although it is just a customary).